keyboards/s_ol/0xc_pad/keymaps/default/readme.md

```diff
@@ -0,0 +1,5 @@
+![0xC.pad Layout Image](https://i.imgur.com/pk4G33B.png)
+​
+# Default 0xC.pad Layout
+​
+This is a simple default layout to get you started with the 0xC.pad.
```

Comment:
Switching this around so the text is first.

```suggestion
# Default 0xC.pad Layout
​
This is a simple default layout to get you started with the 0xC.pad.

![0xC.pad Layout Image](https://i.imgur.com/pk4G33B.png)
​```

Reply:
@noroadsleft I followed the keymap documentation template found [here](https://docs.qmk.fm/#/documentation_templates), is that not up to date?